bento4Config.cmake

```diff
@@ -0,0 +1,2 @@
+include(CMakeFindDependencyMacro)
```

Comment:
This file seems to be by itself and not referenced anywhere

Reply:
It is helpful in case of 3rd party dependencies but indeed it is not needed here